quaireaux/data_structure/cairo_project.toml

```diff
@@ -1,3 +1,3 @@
 [crate_roots]
-data_structures = "src"
+data_structure = "src"
```

Comment:
why no 's'? There are multiple DS

Reply:
Then we should put maths and sortings just feels weird that one in is plural while all the others are singular

Reply:
No, because  the "data-structure**s**" crate defines multiple data **structures**, and pluralizing the name helps convey that there is a collection of them in the crate.

However in the case of "sorting", it's a specific concept that refers to the process of sorting items. While you could use a plural form like "sorting-algorithms" or "sorting-methods", but I think the singular form conveys the idea more efficiently.

just a nitpick but I think it was worth sharing the thought :)

Reply:
I'll revert this change then 😄

Reply:
I wasn't sure about that change, so your concern confirms my doubts 😃